hadoop-hdfs-project/hadoop-hdfs-client/src/main/proto/datatransfer.proto

```diff
@@ -223,6 +225,7 @@ message ShortCircuitShmRequestProto {
   // purely for logging / debugging purposes.
   required string clientName = 1;
   optional DataTransferTraceInfoProto traceInfo = 2;
+  optional bytes spanContext = 3;
```

Comment:
The OT span context should be added into DataTransferTraceInfoProto

Reply:
Done in b8c63f3f5d65e4d94caef98587d468bdbab78e5d (same commit as above protobuf change).